engines/python/setup/djl_python/properties_manager/lmi_dist_rb_properties.py

```diff
@@ -82,6 +86,14 @@ def validate_speculative_and_lora(self):
             )
         return self
 
+    @model_validator(mode='after')
+    def validate_speculative_and_fml(self):
+        if self.load_format == LmiDistLoadFormats.sagemaker_fast_model_loader and self.speculative_draft_model:
```

Comment:
self.load_format seems to be str. So you might need to change to `self.load_format == LmiDistLoadFormats.sagemaker_fast_model_loader.value`.  Or change the load_format type from str to LmiDistLoadFormats enum. I prefer the latter.

Reply:
it seems self.load_format  is used in other places. Let me do the .value change first.